src/i18n/locales/en.json

```diff
@@ -140,6 +140,9 @@
   "video-settings-open": "Open video settings",
   "video-settings-close": "Close video settings",
 
+  "desktop-audio-info": "Desktop audio is recorded",
+  "no-desktop-audio-info": "Desktop audio is not recorded. \n If you want to record desktop audio, please use one of the following browsers: \n - Google Chrome, Edge and Opera: On Windows, the entire system audio can be captured, but on Linux and macOS only the audio of a tab can be captured. \n - Firefox, Safari and Internet Explorer do not support recording desktop audio.",
```

Comment:
> but on Linux and macOS only the audio of a tab can be captured.

I'm not quite sure what this means. What will be recorded?

> Internet Explorer 

I wouldn't add obscure and obsolete browsers to this list.

> Firefox […] do not support recording desktop audio.

Not true. Firefox lets you select all video sources. That includes monitor sources. It's just that the implementation would need to be slightly different.